migrations/2023-01-03-084740_remove-collections-v1/down.sql

```diff
@@ -0,0 +1,137 @@
+-- This file should undo anything in `up.sql`ALTER TABLE settings
```

Comment:
This does not repopulate the v1 `collection_items` table. We don't need that but either implement it or add a comment that this is a migration that _loses_ data.